download_models/requirements.txt

```diff
@@ -0,0 +1,3 @@
+openvino==2024.6.0
+openvino-dev
+ultralytics
```

Comment:
There is one requirements.txt exists in src/ folder. We can add these modules also there and expect user to install them before running the application.